Core/src/org/sleuthkit/autopsy/discovery/search/DiscoveryEventUtils.java

```diff
@@ -88,6 +88,14 @@ public ClearInstanceSelectionEvent() {
             //no arg constructor
         }
     }
+    
+    /**
+     * Event to signal that any background tasks currently running should
+     * be cancelled.
+     */
+    public static final class CancelBackgroundTasksEvent {
+        
```

Comment:
Can we add an empty no arg constructor similar to the ClearInstanceSelectionEvent for this so that netbeans doesn't think it can improve things and tell us 'this may be an interface'?